### PR TITLE
Add Epic Games Store integration

### PR DESCRIPTION
**Pull request by Quentame**
## Proposed change

Add Epic Games Store integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/104725
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/3820
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

File: source/_integrations/epic_games_store.markdown [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+## Sensors
+
+The following sensors will be created :
+
+|Entity|Description|
+|------|-----------|
+|`free_game_1`|First current free game|
+|`free_game_2`|Second current free game|
+|`next_free_game_1`|First upcoming free game|
+|`next_free_game_2`|Second upcoming free game|
```
Comment (Quentame) [inline source/_integrations/epic_games_store.markdown]: Following https://github.com/home-assistant/core/pull/81167#issuecomment-1398180728 the parent's PR changed this behavior.

```suggestion
|`free_games`|Current free games|
|`next_free_games`|Upcoming free games|
```

File: source/_integrations/epic_games_store.markdown [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+title: Epic Games Store
+description: Instructions on how to integrate Epic Games Store service into Home Assistant.
+ha_category:
+  - Sensor
+ha_release: 2022.12
```
Comment (Quentame) [inline source/_integrations/epic_games_store.markdown]:
```suggestion
ha_release: 2024.5
```
🤞

File: source/_integrations/epic_games_store.markdown [hunk 3]
```diff
@@ -0,0 +1,31 @@
+---
+title: Epic Games Store
+description: Instructions on how to integrate Epic Games Store service into Home Assistant.
+ha_category:
+  - Calendar
+ha_release: 2022.12
+ha_iot_class: Cloud Polling
+ha_codeowners:
+  - '@Quentame'
```
Comment (Quentame) [inline source/_integrations/epic_games_store.markdown:10]:
```suggestion
  - '@hacf-fr'
  - '@Quentame'
```

File: source/_integrations/epic_games_store.markdown [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+Integrates [Epic Games Store](https://store.epicgames.com/) service into Home Assistant.
+
+The integration adds calendars to follow [discounts & free games](https://store.epicgames.com/free-games).
+
+{% include integrations/config_flow.md %}
+
+
+## Calendars
+
+The following calendars will be created :
```
Comment (c0ffeeca7) [inline source/_integrations/epic_games_store.markdown]:
```suggestion
The following calendars will be created:
```

File: source/_integrations/epic_games_store.markdown [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+  - '@hacf-fr'
+  - '@Quentame'
+ha_domain: epic_games_store
+ha_config_flow: true
+ha_platforms:
+  - calendar
+ha_integration_type: service
+---
+
+Integrates [Epic Games Store](https://store.epicgames.com/) service into Home Assistant.
```
Comment (c0ffeeca7) [inline source/_integrations/epic_games_store.markdown]:
```suggestion
The **Epic Games Store** {% term integration %} integrates the [Epic Games Store](https://store.epicgames.com/) service into Home Assistant.
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @Quentame :+1:

**Pull request merged**